spec/controllers/tasks_controller_spec.rb

```diff
@@ -252,6 +274,11 @@
         expect { put_update }.to change { task.reload.license.name }.to('new_license')
       end
 
+      it 'updates the tasks labels' do
+        expect(task.labels).to eq([existing_label])
```

Comment:
You don't really need this line, since you are basically testing if the factory works.
If you really want to make sure that `existing_label` is correctly set, you could add a `from` to the other expectation (expect-to change-from-to)